modules/Link.js

```diff
@@ -22,6 +22,10 @@ function isEmptyObject(object) {
   return true
 }
 
+function resolveToLocation(to, router) {
```

Comment:
This isn't a class method because we need to strip out `to` from `props` in `render` anyway – we can't (shouldn't) pass it down because otherwise we get React 15.2.x "unknown props" warnings.

The other options would be to either destructure it out and leave it unused, or do an explicit `delete`, both of which are uglier than just using it as a function arg.